lib/generator/models.ex

```diff
@@ -12,24 +12,41 @@ defmodule Thrift.Generator.Models do
     Enum.flat_map(file_group.schemas, fn {_, schema} ->
       schema
       |> Map.put(:file_group, file_group)
-      |> generate_schema(output_dir)
+      |> generate_schema
+      |> write_schema_to_file(output_dir)
     end)
   end
 
-  def generate_schema(schema, output_dir) do
+  def generate_to_string!(%FileGroup{}=file_group) do
+    Enum.flat_map(file_group.schemas, fn {_, schema} ->
+      schema
+      |> Map.put(:file_group, file_group)
+      |> generate_schema
+    end)
+    |> Enum.reverse
+    |> Enum.map(fn {_, code} ->
+      Macro.to_string(code)
+    end)
+    |> Enum.join("\n")
```

Comment:
What's up with the join? Previously a single .thrift input file would generate multiple output files, one for each struct/enum/etc. Does this change that?